src/app/core/shared/resource-policy.model.ts

```diff
@@ -0,0 +1,20 @@
+
+import { CacheableObject } from '../cache/object-cache.reducer';
+import { ResourceType } from './resource-type';
+
+export class ResourcePolicy implements CacheableObject {
```

Comment:
This class should have comments to describe the various fields.  See, for example, `item.model.ts`